src/Client.php

```diff
@@ -328,12 +328,29 @@ public function getGuzzleClient() {
     return $this->guzzleClient;
   }
 
+  public function filterEmptyArrays($array) {
```

Comment:
missing doc block

Reply:
this would be a great candidate for some tests so that one could see what some of the inputs this is expected to handle.

Reply:
should it be `filterEmptyArrays(array $array)` ?

Reply:
Can't be type array because recursive :(

Reply:
This method shouldn't be public - it's not part of the purpose of this class - just internal functionality that other classes shouldn't rely on.

Reply:
Actually, it will always be an array, so yeah, can do that, and docblock added.